### DISCUSSION TITLE
Profile for Bambulab A1

### DISCUSSION THREAD

**Discussion by alex-12-2023** [OrcaSlicer/OrcaSlicer#3229] [Q&A]
please Profile for Bambulab A1
thanks
![3d-printer-bambu-lab-a1-15-500x500 1](https://github.com/SoftFever/OrcaSlicer/assets/154546276/e13e8e93-c585-4b75-a3ac-5dac02144945)

**Comment by loom4k** [comment]
I suppose it wouldn't be too hard to import a profile coming from BambuStudio. It might just come when the next pull from Studio is made though 😀
